### ISSUE TITLE
Could not find OpenGL or GLUT so simviz will not be built, but both are in place.

### ISSUE THREAD

**Issue by ignatenkobrain** [simit-lang/simit#27]
```
-- Found OpenGL: /usr/lib64/libGL.so  
-- Found GLUT: /usr/lib64/libglut.so  
Could not find OpenGL or GLUT so simviz will not be built
```

No idea why.

**Comment by VikingScientist** [comment]
I had the same issue. Turns out that the compiler needs GLUT Xi and GLUT Xmu in addition to openGL and GLUT. Installing these fixed it for me. On linux ubuntu this is done by typing
```
sudo apt-get install libxmu-dev libxi-dev
```

**Comment by fredrikbk** [comment]
@VikingScientist did the visualizer work for you? As far as I know it hasn't been used for a long time so I wasn't sure whether it had rotted.

**Comment by s1l3n0** [comment]
@VikingScientist +1

**Comment by fulviofarina** [comment]
You need more packages probably:

Try installing these:

**sudo apt-get install freeglut3 freeglut3-dev binutils-gold libeigen3-dev libeigen3-doc libxmu-dev libxi-dev**

and then "cmake .."  again your " build" (you'll probably need to erase the folder and remake it again)


Also, it is possible to install llvm version 3.8 from
**sudo apt-get, install llvm** 
instead of the 3.4 version

**Comment by VikingScientist** [comment]
@fredrikbk Sorry, I haven't gotten around to testing the visualizer yet. I just managed to get it compiling.

**Comment by Lugatod** [comment]
@VikingScientist +1
Worked for me too.

**Comment by fredrikbk** [comment]
I don't believe anyone uses the visualizer (?) and I believe it's bit rotted. It also uses glut which seems to be problematic on new macOS.

Unless anyone uses the current visualizer I will delete it from the repo. We have plans for making a new one and hope to get someone working on it soon.
